pennylane/labs/resource_estimation/templates/qubitize.py

```diff
@@ -30,14 +30,23 @@
 
 
 class ResourceQubitizeTHC(ResourceOperator):
-    r"""Resource class for Qubitization of THC Hamiltonian.
+    r"""Resource class for qubitization of a Hamiltonian.
```

Comment:
We should still specify THC hamiltonian here right? Since the class is called QubitizeTHC